cross-doc-explainer.md

```diff
@@ -0,0 +1,163 @@
+
+The minimal functional opt-in would be a global declaration that the document supports view
+transitions, e.g.:
+
+```html
+<meta name="view-transitions" content="same-origin">
+```
+
+though to make this fully expressive, e.g. opt in conditionally based on reduced-motion preferences,
+versions, or URL patterns, this would need a more elaborate definition, e.g.:
+
+```css
+@cross-document-view-transitions: allow;
+@prefers-reduced-motion {
+   @cross-document-view-transitions: skip;
+}
+```
+
+Note: The exact semantics of the conditional opt-in are TBD. See related open issues:
+* w3c/csswg-drafts#8048
+* w3c/csswg-drafts#8679
+* w3c/csswg-drafts#8683
+
+## Lifecycle
+
+![Lifecycle chart for cross-document transitions](media/mpa-chart.svg)
+
+### Capturing the old state
+
+[The old state is captured](https://drafts.csswg.org/css-view-transitions-1/#capture-old-state-algorithm) right before the old document is hidden and a new one is shown.
+In the HTML spec that moment is defined [here](https://html.spec.whatwg.org/#populating-a-session-history-entry:loading-a-document).
+This can either happen during normal navigations, when the new document is about to be created,
+in Back/Forward cache navigations, or when activating a prerendered document.
+
+Before creating the new document (or activating a cached/prerendered one), the UA would [update the rendering](https://html.spec.whatwg.org/#update-the-rendering) and snapshot the old document, in the same manner a document is snapshotted for a same-document navigation.
+
+The developer can use existing events like `navigate` (where available) or `click` to customize the
+elements which have a view-transition-name in the old Document.
+
+Example:
+```js
+navigation.addEventListener("navigate", event => {
+  // Don't capture navigation-bar animation when navigating to home
+  if (!new URL(event.destination.url).pathname.startsWith("/home"))
+    navigationBar.viewTransitionName = "none";
+});
+```
+
+### Capturing the new state
+
+The [new state is captured](https://drafts.csswg.org/css-view-transitions-1/#capture-new-state-algorithm) right before the first [rendering opportunity](https://html.spec.whatwg.org/#rendering-opportunity)
+of the new document. This allows the new document to use the
+[render-blocking mechanism](https://html.spec.whatwg.org/#render-blocking-mechanism) as a way to
+delay the transition.
+
+As shown in the chart above, that first rendering opportunity can come in two cases, either
+it's a newly initialized document that's no longer [render-blocked](https://html.spec.whatwg.org/multipage/dom.html#render-blocked), or it's a document that's been frozen due to back-forward cache
+or prerendered, and is now being activated.
+
+Note: relying on the [render-blocking](https://html.spec.whatwg.org/multipage/dom.html#render-blocked) mechanism is limited, as only a limited set of elements participate in that
+mechanism. See [proposal to extend it](whatwg/html#9332).
```

Comment:
```suggestion
mechanism. See [proposal to extend it](https://github.com/whatwg/html/issues/9332).
```

Reply:
Is that necessary? Github links work magically in Github MarkDown